CoffeeChat/Controller/EditDemographicsViewController.swift

```diff
@@ -227,7 +227,7 @@ class EditDemographicsViewController: UIViewController {
 
 extension EditDemographicsViewController: OnboardingDropdownViewDelegate {
 
-    func updateSelectedFields(tag: Int, isSelected: Bool) {
+    func updateSelectedFields(tag: Int, isSelected: Bool, valueSelected: String) {
         fieldsEntered[tag] = isSelected
```

Comment:
Unless this is all the function is intended to do, could you add a `TODO` or fill it out? I imagine it's pretty similar to the one in `DemographicsViewController`.